doc/install/index.rst

```diff
@@ -1,72 +1,74 @@
-=======================
- Installation (Manual)
-=======================
+.. _install-overview:
 
+===============
+Installing Ceph
+===============
 
-Get Software
+There are different options to install Ceph. Each solution meet different
+requirements, like ease of maintenance.
```

Comment:
Each method tends to meet different requirements, like ease of maintenance, and many solutions are better suited for different versions.  Please see the notes associated with each method before choosing.

Reply:
Each method meets different requirements (for instance, some result in Ceph clusters that are easier to maintain). Review the documentation for each method before choosing one.

Reply:
do you have an idea of how to accomplish this? A table? Just extend each sections with a list of caveats + requirements?